test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should not be pushed to the remote repo since it's listed in `.gitignore`. This issue could be because you committed the file before putting it in `.gitignore`, you can fix this by untracking the file. Kindly check [the following link](https://sufiyanyasa.com/blog/git-untrack-file/) for more info.